resolwe/flow/executors/prepare.py

```diff
@@ -0,0 +1,49 @@
+"""Base class for the manager-internal preparation bits of the executor."""
+from __future__ import absolute_import, division, print_function, unicode_literals
+
+import logging
+import os
+
+from django.apps import apps
+from django.conf import settings
+from django.forms.models import model_to_dict
+
+from resolwe.flow.models import Data
+
+logger = logging.getLogger(__name__)  # pylint: disable=invalid-name
+CWD = os.getcwd()
```

Comment:
Where is this used and why keep it in a global variable?

Reply:
This is an really old variable and can safely be removed.